README.md

```diff
@@ -32,15 +32,17 @@ navigate to `benchmarks/`. Benchmarks data is organized by performance year.
 For example, `benchmarks/2017.json` contains the benchmarks for performance year 2017
 (benchmark year 2015).
 
+$YEAR refers to the performance year. For measures data, providing a $YEAR is optional. If omitted, it defaults to 2018.
+
 ### Importing the qpp-measures-data module
 Functions take a string version argument and return the appropriate YAML schema or JSON data.
 The module can be used with the following pattern:
 ```javascript
 const qppMeasuresData = require('qpp-measures-data');
-const measuresData = qppMeasuresData.getMeasuresData(2018);
-const measuresSchema = qppMeasuresData.getMeasuresSchema(2018);
+const measuresData = qppMeasuresData.getMeasuresData($YEAR);
+const measuresSchema = qppMeasuresData.getMeasuresSchema($YEAR);
 const benchmarksData = qppMeasuresData.getBenchmarksData();
-const benchmark2017Data = benchmarksData[2018];
+const benchmark2017Data = benchmarksData[$YEAR];
```

Comment:
Maybe this variable should be renamed?

Reply:
@harrisj haha good catch. i'll fix it in my upcoming pr